src/middlewared/middlewared/plugins/docker_linux/images.py

```diff
@@ -0,0 +1,32 @@
+import aiodocker
+import os
+
+from middlewared.service import CallError, private, CRUDService
+
+
+DEFAULT_DOCKER_IMAGES_PATH = '/usr/local/share/docker_images/docker-images.tar'
+
+
+class DockerImagesService(CRUDService):
+
+    class Config:
+        namespace = 'docker.images'
```

Comment:
Is this really supposed to be a public service?

Reply:
Yes, we build on this service for the complete docker images part which we want to expose in the UI